hosting/setup-guides/linux.md

```diff
@@ -167,10 +312,6 @@ Alias=hostd.service
 
 You can now save the file with `ctrl+s` and exit with `ctrl+x`.
 
-{% hint style="warning" %}
-If you are planning on using the zen testnet, make sure to change the `-http` flag to port `:9880`
```

Comment:
This is unnecessary.

Also since it won't let me add a comment on it: `address: :9880` in the YAML example is unnecessary. The minimum YAML file is:
```yaml
recoveryPhrase: asdasd
http:
  password: asdasd
```